javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,82 @@
+// Flight booking fullname function
+
+function getFullname(firstname, surname, gender, useFormalName) {
+  if (gender === "man" && useFormalName === true) {
+    console.log("Your client full name is lord " + firstname + " " + surname);
+  } else if (gender === "woman" && useFormalName === true) {
+    console.log(
+      "Your client full name is princess " + firstname + " " + surname
+    );
+  } else {
+    console.log("Your client full name " + firstname + " " + surname);
+  }
+}
+let useFormalName = true;
+getFullname(
+  prompt("please enter your first name"),
+  prompt("please enter your last name"),
+  prompt("what is your gender ?"),
+  true
+);
+
+// Event application
+
+let today = new Date();
+let dayToEvent = +prompt("How Many Days Left To The Event ?");
+function getEventWeekday(days) {
+  today.setDate(today.getDate() + days);
+  return today.toLocaleString("default", { weekday: "long" });
```

Comment:
nice work figuring out how to work with date.

Reply:
Thanks, honestly i got lucky not sure how exactly its working XD